docs/ReleaseProcess.md

```diff
@@ -243,7 +243,23 @@ If you encounter an error mentioning:
 found that downgrading `setuptools` to a version `< 66` seems to solve this
 problem.
 
-# Steps for cutting a release
+# Adding a changelog fragment
+
+When a given branch contains a change that deserves a changelog entry, you
+should add a changelog
+[fragment](https://scriv.readthedocs.io/en/stable/concepts.html#fragments). When
+we [cut a release](#cutting-a-release) the changelog fragments will be merged
+into a changelog update. To add a changelog fragment, use `scriv`. See [this
+section](#installing-scriv) for instructions on how to install it.
+
+1. Run `scriv create` on the directory of the package you want to create an
+   entry for (eg `ouroboros-consensus` or `ouroboros-consensus-cardano`). This
```

Comment:
Every package is symlinked (is this true for `tutorials` too?) So running `scriv` in `ouroboros-consensus-test` will put the entry in `ouroboros-consensus/changelog.d`. This is so that in each bundle, no matter which package you have modified, `scriv create` will create it in the right place

Reply:
What about this https://github.com/input-output-hk/ouroboros-consensus/pull/10/commits/d1249fe4ffed33096f7bffa2636dfea002521511